mscore/revision.h

```diff
@@ -1 +1 @@
-3543170
+57d3883
```

Comment:
regarding undo.cpp: 
I think the code I commented out, should be there to not show fingerings in linked staves. Currently I cannot add any fingerings in such a tablature stave, if the option is not set, but they will be shown, if they are set in the notation system ... This shouldn't be at all, as this problem was reason for disabling fingerings in tablature lines.